internal/clearingway/handlers.go

```diff
@@ -466,6 +518,85 @@ func (c *Clearingway) RemoveAll(s *discordgo.Session, i *discordgo.InteractionCr
 	}
 }
 
+func (c *Clearingway) RequestReclear(s *discordgo.Session, i *discordgo.InteractionCreate) {
```

Comment:
now that i understand the part about toggling, let's reflect the name to be just so: `ToggleReclear`